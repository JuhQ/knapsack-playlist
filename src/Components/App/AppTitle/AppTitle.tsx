import React from "react";
import { Container } from "semantic-ui-react";

const AppTitle: React.FC = () => (
  <Container textAlign="center">
    <h1 data-text="Playlist Generator">Playlist Generator</h1>
    <span className="App-logo" role="img" aria-label="rock on!">
      ðŸŽ¸
    </span>
    <span className="App-logo" role="img" aria-label="rock on!">
      ðŸ¤˜
    </span>
    <br />
    <small>
      Select either{" "}
      <code>
        <strong>Option 1</strong>
      </code>{" "}
      or{" "}
      <code>
        <strong>Option 2</strong>
      </code>
      .
    </small>
  </Container>
);

export default AppTitle;
