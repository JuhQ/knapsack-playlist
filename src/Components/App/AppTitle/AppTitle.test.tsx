import React from "react";
import { create } from "react-test-renderer";

import AppTitle from "./AppTitle";

describe("AppTitle", () => {
  it("should render", () => {
    const component = create(<AppTitle />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
