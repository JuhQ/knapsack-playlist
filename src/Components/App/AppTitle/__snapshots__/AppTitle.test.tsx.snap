// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle should render 1`] = `
<div
  className="ui center aligned container"
>
  <h1
    data-text="Playlist Generator"
  >
    Playlist Generator
  </h1>
  <span
    aria-label="rock on!"
    className="App-logo"
    role="img"
  >
    ðŸŽ¸
  </span>
  <span
    aria-label="rock on!"
    className="App-logo"
    role="img"
  >
    ðŸ¤˜
  </span>
  <br />
  <small>
    Select either
     
    <code>
      <strong>
        Option 1
      </strong>
    </code>
     
    or
     
    <code>
      <strong>
        Option 2
      </strong>
    </code>
    .
  </small>
</div>
`;
